workflows/iuf/operations/extract-release-distributions.yaml

```diff
@@ -162,10 +162,31 @@ spec:
           #!/bin/sh
           set -x
           media_dir={{inputs.parameters.media_dir}}
+          tar_filename={{inputs.parameters.tarfile}}
+          echo "DEBUG Inputs for extract-tar-files are - tarfile=$tar_filename, media_dir=$media_dir"
+
           cd $media_dir
-          dir_name=`tar -tzf {{inputs.parameters.tarfile}} | head -1 | cut -f1 -d"/"`
-          tar -xvzf {{inputs.parameters.tarfile}} &> /dev/stderr
-          cat ${dir_name}/iuf-product-manifest.yaml > /tmp/manifest.yaml
+          echo "INFO Extracting product tarball $media_dir/$tar_filename"
+          result=$(gunzip -c $tar_filename | tar -t 2>&1)
+          if [ $? -ne 0 ]; then
+            echo "ERROR Provided product tarfile $tar_filename is corrupted"
+            echo "NOTICE Please replace with the valid product tarfile and re-run this 'process-media' stage"
```

Comment:
because action is tied to the error, this should probably be ERROR as well

Reply:
Addressed